src/components/bcros/filing/CommonTemplate.vue

```diff
@@ -77,6 +77,16 @@
         </template>
       </slot>
 
+      <slot name="document-record">
+        <!-- if we have document records(aka filings staff completed on behalf of a client), show them -->
+        <template
+          v-if="!isStaffFiling(filing) && enableDocumentRecords && !!getDocIdByFilingId(documents, filing.filingId)"
+        >
+          <UDivider class="my-6" />
+          <BcrosDocumentRecordBtn :document-id="getDocIdByFilingId(documents, filing.filingId)" />
```

Comment:
You call this function twice. How about making it a computed?

Reply:
Updated